draft-song-dns-wireformat-http-01.txt

```diff
@@ -312,37 +315,49 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
    bytes length field in DNS over TCP.
 
    Since this two-byte length field is redundant, when the client proxy
-   receives a DNS message over TCP, it does NOT include the length field
+   receives a DNS message over TCP, it MUST NOT include the length field
    in the message sent to the server.  The length in the content-length
-   header is only the size of the DNS message itself, and does NOT
+   header is only the size of the DNS message itself, and MUST NOT
    include this two-byte length header.
 
-4.  Considerations on DNS over HTTPS
+4.  Security Considerations
 
-   HTTPS is recommended for communication.  It provides privacy and
-   integrity for HTTP sessions.
+   Since this protocol transmits DNS messages, all of the security
+   concerns that stub resolvers and recursive resolvers have with DNS
+   messages apply.  However, since HTTP uses TCP as the underlying
```

Comment:
"all of the security concerns that stub resolvers and recursive resolvers have with DNS massage apply"-- for audience without much knowledge of previous backgroud discussion security concerns on DNS , can we provides some reference pointing to the security section of other RFC? Is it necessary? Because I feel it is uninformative for me.